lib/src/Datepicker/Datepicker.tsx

```diff
@@ -239,6 +260,10 @@ export const Datepicker: React.FunctionComponent<DatepickerProps> = React.forwar
         const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
             const { value: changeEventValue } = e.target;
             const value: Date = new Date(changeEventValue);
+            if (min > value || max < value) {
+                onChange(null);
+                return;
+            }
             onChange(value);
```

Comment:
is there a requirement to trigger `onChange` with null value? if there isn't maybe we can tap into existing helper method and trigger event only when they are valid?

```
if (isDateInRange(value, min, max)) {
  onChange(value);
}
```

Reply:
updated